contracts/test/forkMainnet/AMMWrapper/SetupTest.t.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+
+import "contracts-test/forkMainnet/AMMWrapper/Setup.t.sol";
+
+contract TestAMMWrapperSetup is TestAMMWrapper {
```

Comment:
I think this could be embedded into `TestAMMWrapper`

Reply:
Thanks @oneleo 's reminder, I didn't aware that it will cause sub test suite to repetitively run these tests.

Reply:
Because the contract inherits from Setup.t.sol, if it is embedded into TestAMMWrapper.t.sol, it will result in multiple test outputs, like below:

<details>
  <summary>( Please click here to expand )</summary>
    <pre><code>

Running 4 tests for contracts/test/forkMainnet/AMMWrapper/TradeSushiswap.t.sol:TestAMMWrapperTradeSushiswap
**[PASS] testAMMWrapperSetup() (gas: 60516)
[PASS] testTokensAllowanceAmountWhenSetup() (gas: 34022)
[PASS] testTokensTotalSupplyWhenSetup() (gas: 25809)**
[PASS] testTradeSushiswap() (gas: 262643)
Test result: ok. 4 passed; 0 failed; finished in 1.41s

Running 4 tests for contracts/test/forkMainnet/AMMWrapper/TradeCurveV1.t.sol:TestAMMWrapperTradeCurveV1
**[PASS] testAMMWrapperSetup() (gas: 60538)
[PASS] testTokensAllowanceAmountWhenSetup() (gas: 34044)
[PASS] testTokensTotalSupplyWhenSetup() (gas: 25831)**
[PASS] testTradeCurveV1() (gas: 513332)
Test result: ok. 4 passed; 0 failed; finished in 1.42s

Running 7 tests for contracts/test/forkMainnet/AMMWrapper/TradeUniswapV2.t.sol:TestAMMWrapperTradeUniswapV2
**[PASS] testAMMWrapperSetup() (gas: 60604)
[PASS] testTokensAllowanceAmountWhenSetup() (gas: 34045)
[PASS] testTokensTotalSupplyWhenSetup() (gas: 25810)**
[PASS] testCannotTradeIfPayloadAgain() (gas: 286983)
[PASS] testCannotTradeWithInvalidSignature() (gas: 77447)
[PASS] testTradeUniswapV2() (gas: 254291)
[PASS] testTradeWithOldEIP712Signature() (gas: 254271)
Test result: ok. 7 passed; 0 failed; finished in 1.42s
    </code></pre>
</details>

Sorry about that...